### PR TITLE
NL: Add more per-chart emojis

### PR DESCRIPTION
**Pull request by pradh**
Intended meaning:

1. "This chart is not relevant for the query" -- 👎
2. "Somewhat relevant but not quite right (eg wrong place type)" -- ⚠️
3. "chart should be promoted" -- ⬆️
4. "chart should be demoted" -- ⬇️
5. "god, this is so embarrassing" -- 🤦

Example:

image

### CODE REVIEW

File: static/js/apps/nl_interface/query_result.tsx [hunk 1]
```diff
@@ -149,7 +149,9 @@ export const QueryResult = memo(function QueryResult(
           </a>
           {chartsData && chartsData.sessionId && (
             <span
-              className={`thumb-down ${isThumbClicked ? "thumb-dim" : ""}`}
+              className={`feedback-emoji ${
+                isThumbClicked ? "feedback-emoji-dim" : ""
```
Comment (juliawu) [inline static/js/apps/nl_interface/query_result.tsx:153]: Is the intended behavior for query_result to be specific to thumbs down?

If not, let's use `isEmojiClicked` instead and update the function on line 200 to onChartClick to match nl_feedback.tsx

File: static/js/utils/nl_interface_utils.ts [hunk 2]
```diff
@@ -30,6 +30,10 @@ const DEBUG_INFO_PARAM_PREFIX = "&entry.1280679042=";
 
 export const CHART_FEEDBACK_SENTIMENT = {
   THUMBS_DOWN: "THUMBS_DOWN",
+  WARNING: "WARNING",
```
Comment (juliawu) [inline static/js/utils/nl_interface_utils.ts:33]: nit: let's add quick comments here explaining what feedback each emoji means

**Review by juliawu — COMMENTED** [review]
Thanks!

File: static/js/apps/nl_interface/query_result.tsx (hunk 1 above)
Comment (pradh) [reply static/js/apps/nl_interface/query_result.tsx:153]: Yes, thumbs-down for now.

File: static/js/utils/nl_interface_utils.ts (hunk 2 above)
Comment (pradh) [reply static/js/utils/nl_interface_utils.ts:35]: Done.

**Review by pradh — COMMENTED** [review]
Thanks for the review!

**Review by juliawu — APPROVED** [review]
Thanks for the changes!

**Pull request merged**